lib/openstax/aws/rds_instance.rb

```diff
@@ -0,0 +1,26 @@
+module OpenStax::Aws
+  class RdsInstance
+
+    attr_reader :raw
+
+    delegate_missing_to :@raw
+
+    def initialize(db_instance_identifier:, region:)
+      @raw = Aws::RDS::DBInstance.new(
+        db_instance_identifier,
+        client: Aws::RDS::Client.new(region: region)
+      )
+    end
+
+    def set_rds_master_password( db_instance_identifier: ,password:)
+      raw.modify_rds_instance({
```

Comment:
I think this method is `modify` instead of `modify_rds_instance`, and it doesn't take a `db_instance_identifier` (which makes sense because the `raw` object already knows the identifier).